layouts/partials/footer.html

```diff
@@ -5,62 +5,54 @@
           {{ readFile "assets/maxmind-footer-logo.svg" | safeHTML }}
         </a>
         <div class="footer__social">
-          <a aria-label="Twitter" class="footer__socialIcon" href="https://twitter.com/maxmind">
+          <a aria-label="x" class="footer__socialIcon" margin-top: -.15em href="https://x.com/maxmind">
```

Comment:
@AndraMeer @dlieou Just a heads up for future reference, we shouldn't add in inline styles. We should be adding in styles via the `scss` sheet.